docs/admin/scale/index.md

```diff
@@ -0,0 +1,69 @@
+We scale-test Coder with the [same utility](#scaletest-utility) that can be used in your environment for insights into how Coder scales with your infrastructure.
+
+## General concepts
+
+Coder runs workspace operations in a queue. The number of concurrent builds will be limited to the number of provisioner daemons across all coderd replicas.
+
+- **coderd**: Coder’s primary service. Learn more about [Coder’s architecture](../../about/architecture.md)
+- **coderd replicas**: Replicas (often via Kubernetes) for high availability, this is an [enterprise feature](../../enterprise.md)
+- **concurrent workspace builds**: Workspace operations (e.g. create/stop/delete/apply) across all users
+- **concurrent connections**: Any connection to a workspace (e.g. SSH, web terminal, `coder_app`)
+- **provisioner daemons**: Coder runs one workspace build per provisioner daemon. One coderd replica can host many daemons
+- **scaletest**: Our scale-testing utility, built into the `coder` command line.
```

Comment:
nit: People who are not familiar with Coder internals may find it hard to imagine it. You can consider adding a picture later. I know that they can skim through linked documents, but it saves a few minutes of life :)

Reply:
would love to do this down the road.